tests/unit_test.R

```diff
@@ -0,0 +1,38 @@
+library(kaiaulu)
+library(testthat)
+
+test_that("calling parse_gitlog with correct perceval path and an exist local path of github project", {
```

Comment:
This is a very good start for the description. It has a good amount of detail, but it is missing the behavior on the sentence. "test that calling parse_ghitlog with correct perceval path **returns a data table**). 

(Note I specified data.table, since that's the object parse_gitlog returns, using the data.table package, which is different than R's native dataframe).

Reply:
Please apply the fix to the description to the subsequent tests too.